client/my-sites/checkout/checkout/composite-checkout.js

```diff
@@ -315,3 +319,16 @@ function CountrySelectMenu( {
 		</FormFieldAnnotation>
 	);
 }
+
+function TestingBanner() {
+	return (
+		<Card
+			className="composite-checkout__testing-banner"
+			highlight="warning"
+			href="https://github.com/Automattic/wp-calypso/issues/new"
+		>
+			Warning! This checkout form is a new feature, still in testing. If you encounter issues,
```

Comment:
I don't think we need the comma after feature. 

`This checkout is a new feature still in testing`  sounds right to me.